platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/JewelConfigs.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.jewel.foundation
+
+@InternalJewelApi
+public object JewelConfigs {
+    public val useCustomPopupRender: Boolean
```

Comment:
Please add KDoc to this property explaining what it does, and how to enable it from the command line too.